orangecontrib/prototypes/widgets/ow1ka.py

```diff
@@ -255,7 +268,10 @@ def apply_domain_edit(self):
 
     def table_from_html(self, html):
         soup = BeautifulSoup(html, 'html.parser')
-        html_table = soup.find_all('table')[-1]
+        try:
+            html_table = soup.find_all('table')[-1]
+        except IndexError:
+            raise DataEmptyError
```

Comment:
I'd add (). We usually raise Exception instances, not classes.

Reply:
I prefer it this way when the exception has no arguments. It's sugar. https://docs.python.org/3/tutorial/errors.html#raising-exceptions Regex `raise \w+$` shows our use of it is quite broad.

Reply:
I love the "one obvious way to do it" approach python has :) Leave it as it is then